src/gsy_e/models/strategy/scm/smart_meter.py

```diff
@@ -53,7 +53,10 @@ def get_energy_to_buy_kWh(self, time_slot: DateTime) -> float:
     def decrease_energy_to_sell(
             self, traded_energy_kWh: float, time_slot: DateTime, area: "CoefficientArea"):
         """Decrease traded energy from the state and the strategy parameters."""
-        self.state.decrement_energy_requirement(traded_energy_kWh, time_slot, area.name)
+        self._energy_params.decrement_energy_requirement(
+            energy_kWh=traded_energy_kWh,
+            time_slot=time_slot,
+            area_name=area.name)
```

Comment:
This is also changing the behaviour as in [this comment](https://github.com/gridsingularity/gsy-e/pull/1523/files#r963795487).

The previous code was decrementing energy using kWh units, but the `decrement_energy_requirement()` method in the State class expects Wh units instead.